corehq/apps/es/users.py

```diff
@@ -58,6 +58,7 @@ def builtin_filters(self):
             is_practice_user,
             role_id,
             is_active,
+            is_active_in_domain,
```

Comment:
We should also try to combine these two filters into one, so it works regardless of user type.  The `location` and `role_id` filters do similarly.

Reply:
I think I'm getting a bit confused with how these filters typically stack. If `active` and `is_active_in_domain` were combined similarly to how `location` is handled, on its own, that'll return both mobile workers and web users that are active/deactivated, right? So is it expected that the full query should always look like `UserES().web_users().active(True)` / `UserES().mobile_users.active(True)` to limit it to the doc_types we're trying to query for?

Reply:
yes, I think so - otherwise `.active()` implicitly filters to mobile workers, and `is_active_in_domain` likewise filters to web users.

In my opinion, `UserES().active(True)` should return all active users, of both types.  If you want to limit to a particular user type, then you can always add a separate filter for that.